.gitignore

```diff
@@ -14,6 +14,7 @@ node_modules/
 
 # misc
 .DS_Store
+.env
```

Comment:
This may be unnecessary?  Does line 27 in the .gitignore not ignore your .env?

Reply:
Yeah that line looks unnecessary, I forgot about line 27 when i first made it. Resolved